src/lumigo_tracer/lumigo_utils.py

```diff
@@ -84,6 +84,12 @@ class Configuration:
     max_entry_size: int = DEFAULT_MAX_ENTRY_SIZE
     get_key_depth: int = DEFAULT_KEY_DEPTH
 
+    @staticmethod
+    def get_max_entry_size(has_error: bool = False) -> int:
+        if has_error:
+            return Configuration.max_entry_size * 2
```

Comment:
don't you prefer that the `2` will be configurable as well?
At least let it be a constant :-)